src/System.Net.Sockets/src/System.Net.Sockets.csproj

```diff
@@ -1,4 +1,5 @@
-﻿<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Yeah, this change came from some automatic tool that decided by itself to make it. We'll take the original file and only change the new line so next iteration is cleaner.